tasmota/support_tasmota.ino

```diff
@@ -203,8 +203,15 @@ void ZeroCrossMomentEnd(void) {
 
 void IRAM_ATTR ZeroCrossIsr(void) {
   uint32_t time = micros();
-  TasmotaGlobal.zc_interval = ((int32_t) (time - TasmotaGlobal.zc_time));
-  TasmotaGlobal.zc_time = time;
+  static bool lastValue;
+  bool value = digitalRead(Pin(GPIO_ZEROCROSS));
+  if(value && !lastValue) {
```

Comment:
Why this?

The ISR is called on a CHANGE of GPIO so this should ALWAYS be true.